crates/sui-core/src/authority/authority_store_tables.rs

```diff
@@ -70,8 +66,6 @@ pub struct AuthorityPerpetualTables {
     /// objects are still accessible!
     pub(crate) objects: DBMap<ObjectKey, StoreObjectWrapper>,
 
-    pub(crate) indirect_move_objects: DBMap<ObjectContentDigest, StoreMoveObjectWrapper>,
```

Comment:
I think we can't remove tables right now? (I see other DBMaps in those files are just marked with `#[dead_code]` instead?)

Reply:
No, it should be ok. It should be covered by [populate_missing_cfs](https://github.com/MystenLabs/sui/blob/35c9a886f5c81511aa415d9b5518d59b5eddce38/crates/typed-store/src/rocks/mod.rs#L2870). 
Some other deprecated column families are still accessible under certain flags

Reply:
big if true! I could have sworn this didn't work though?

Reply:
ok, I'll cherry-pick and test it on a vultr node, but it should work

Reply:
yeah, it worked